### DISCUSSION TITLE
How conciliate cargo loco start and pnpm dev

### DISCUSSION THREAD

**Discussion by geraldodev** [loco-rs/loco#817] [Q&A] [answered]
Hi,

I'm using a loco new bootstrapped  app that used Client (configures assets for frontend serving)

pnpm dev is running Rsbuild and it wipes dist folder which is needed by cargo loco start.

Error: Message("one of the static path are not found, Folder `frontend/dist` fallback: `frontend/dist/index.html`")

Is possible to run loco with with pnpm dev (without a dist folder) ?

Regards,

Geraldo Lopes de Souza

**Comment by kaplanelad** [comment, marked as answer]
To run loco with pnpm dev without requiring a dist folder, you can follow these steps:

1. Create a new project with the following selections:

  ```
❯ loco new
✔ ❯ App name? · test
✔ ❯ What would you like to build? · SaaS app (with DB and user auth)
✔ ❯ Select a DB Provider · Sqlite
✔ ❯ Select your background worker type · Async (in-process tokio async tasks)
✔ ❯ Select an asset serving configuration · Client (configures assets for frontend serving)
```
2. Navigate to the `frontend `folder and install the dependencies:
  ```
  cd ./test/frontend
pnpm install
```
3. Open the `test/frontend/rsbuild.config.ts` file and ensure that the target API points to the correct loco server endpoint.
4. Finally, run pnpm dev to start the development server

By following these steps, you should be able to run pnpm dev without the need for the dist folder, as it will be dynamically handled during development.

Let me know if you need any help with any issues.
